webboot/webboot.go

```diff
@@ -104,13 +115,12 @@ func name(URL string) (string, error) {
 }
 
 func usage() {
-	log.Printf("Usage: %s [ARGS] URL or name of ISO\n", os.Args[0])
+	log.Printf("Usage: %s [ARGS] URL or name of OS\n", os.Args[0])
```

Comment:
you can do a new PR but I just realized ... [ARGS] is the wrong word. it should be [flags]. An arg is the URL or the bookmark. The other things are flags.

Reply:
Okay,  I can handle this in another PR :)